metrics/line-numbers/lineNumbers_test.go

```diff
@@ -0,0 +1,29 @@
+package linenumbers
+
+import (
+	"testing"
+)
+
+//TestFindLineNumbers runs unit tests on the linenumbers package
+func TestFindLineNumbers(t *testing.T) {
+	keys := []string{"paths", "/alerts?lat={lat}\u0026lon={lon}", "get", "parameters", "1", "name"}
+	token := "lon_100"
+	file := "../../examples/v2.0/yaml/weatherbit.yaml"
```

Comment:
I'd like to avoid including third-party (or at least non-Google) API descriptions in the gnostic repo - ownership and usage rights are unclear. Could you modify this test to instead use the petstore.yaml specs that are already in the repo (in the examples directory)? If so, please also remove weatherbit.yaml from this PR.